test/unit/number/parse.js

```diff
@@ -78,13 +78,18 @@ QUnit.test( "should parse zero-padded decimals", function( assert ) {
 	assert.equal( parse( "0.10", properties( "0.00", en ) ), 0.1 );
 });
 
+QUnit.test( "should parse non-padded decimals", function( assert ) {
+	assert.equal( parse( ".14159", properties( "0.0", en ) ), 0.14159 );
+	assert.equal( parse( ".752", properties( "0.0", en ) ), 0.752 );
+});
```

Comment:
You can `Globalize("ar").formatNumber(0.14159);` :smile: (considering you have loaded data for this locale)